package.json

```diff
@@ -45,17 +45,17 @@
     "cypress-axe": "^0.12.0",
     "cypress-plugin-snapshots": "^1.4.4",
     "danger": "10.5.0",
-    "eslint": "7.11.0",
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-prettier": "^6.13.0",
-    "eslint-plugin-cypress": "^2.11.2",
+    "eslint": "8.1.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-flowtype": "^6.1.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.4.0",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-react": "^7.21.3",
-    "eslint-plugin-react-hooks": "^4.1.2",
+    "eslint-plugin-flowtype": "^8.0.3",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jest": "^25.2.2",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-react-hooks": "^4.2.1-alpha-255221c86-20211103",
```

Comment:
We'll have to use this alpha release for now as the latest stable release isn't compatible with eslint 8.x.x. Hopefully the next stable release happens sometime soon

Reply:
Should we add a slackbot reminder or something to check on this in a month or so?